btax/parameters.py

```diff
@@ -68,6 +68,18 @@ def translate_param_names():
 	# btax_depr_allyr_gds_Switch
 	# btax_depr_allyr_tax_Switch
 
+	class_list = [3, 5, 7, 10, 15, 20, 25, 27.5, 39]
+
+	user_bonus_deprec={'3':btax_depr_3yr_exp, '5':btax_depr_5yr_exp,
+					'7':btax_depr_7yr_exp, '10':btax_depr_10yr_exp,
+					'15':btax_depr_15yr_exp, '20':btax_depr_20yr_exp,
+					'25':btax_depr_25yr_exp, '27.5':btax_depr_27_5yr_exp,
+					'39':btax_depr_39yr_exp}
+	user_deprec_system={'3':btax_depr_3yr_exp, '5':btax_depr_5yr_exp,
```

Comment:
@PeterDSteinberg - Yes, I was going to change somethings about how bonus_deprec is passed since we set up the UI allowing one to enter it separately by asset class.  I've been just working during some meetings so this isn't complete.  Back at it full time now and I'll hope get this running this afternoon.